### ISSUE TITLE
Perf updates based on Lenster PRs to Inbox web

### ISSUE THREAD

**Issue by yash-luna** [xmtp-labs/xmtp-inbox-web#118]
```[tasklist]
### Tasks
- [ ] https://github.com/xmtp-labs/xmtp-inbox-web/issues/213
- [ ] https://github.com/xmtp-labs/xmtp-inbox-web/issues/243
- [ ] Code samples for Perf Tutorial
```

**Comment by yash-luna** [comment]
Reference implementation for message caching once message serialization is ready: https://github.com/xmtp/xmtp-js/issues/291

**Comment by bhavya2611** [comment]
@neekolas @yash-luna I believe this is now a part of the SDK, so we should close this. Reopen if we still need to work on this

**Comment by yash-luna** [comment]
Conversation caching is a part of SDK, afaik message caching isn't. Will let @neekolas confirm

**Comment by neekolas** [comment]
The only thing the SDK provides for message caching is `decodedMessage.toBytes()` and `fromBytes()`. This ticket is for a reference implementation of caching using those functions.

**Comment by yash-luna** [comment]
Is this a dupe of this item - https://github.com/xmtp-labs/xmtp-inbox-web/issues/243 @bhavya2611?

**Comment by yash-luna** [comment]
Added those 2 items as sub-tasks in the issue description

**Comment by rygine** [comment]
@yash-luna closing this in favor of #213. the perf tutorial code samples are covered in https://github.com/xmtp-labs/hq/issues/1002
